src/main/webapp/app/shared/util/string-util.ts

```diff
@@ -0,0 +1,9 @@
+export const concatFullName = (firstName?: string, lastName?: string) => {
+  if (firstName === undefined) {
+    firstName = '';
+  }
+  if (lastName === undefined) {
+    lastName = '';
+  }
+  return firstName + ' ' + lastName;
+};
```

Comment:
Shouldn't throw error if first and last name doesn't exist?

Reply:
Can do that also
But we need to add an catch exception when consume this function
@LUXIANZE you think is needed?

Reply:
Better this way, so that we will not have empty name and other developer can prevent mistake like these.

Reply:
but if first name and last name does not exist server will throw error? If the server will throw error can come into this page?

Reply:
resolved in 64f2e1f0dc946515016105ce79b1f131146ef918